### PR TITLE
Add theme and theme based coloring

### PR DESCRIPTION
**Pull request by jazeee**
Add mixins for theme support.
Convert colors to theme based coloring.
Add theme and theme based coloring
Fix tabs and a few colors
Adjust dark mode button colors.
Update background image
image
image
image
image

Light mode
image
image

### CODE REVIEW

File: ui/colors.scss [hunk 1]
```diff
@@ -1,16 +1,84 @@
...
+    selected-tab-background-color: #b56632,
+    unselected-tab-color: #eee,
+    unselected-tab-background-color: #666,
+    table-row-underline-color: color.scale(#b56632, $alpha: -30%),
+  ),
+);
+
+// Based on https://berbaquero.com/posts/2023/03/sass-dark-mode/
+@mixin theme-mode {
+  body.sheet-darkmode & {
```
Comment (jazeee) [inline ui/colors.scss:49]: This basically adds a css selector: `body.sheet-darkmode & ..... our selectors`, as seen in the compiled file.

This turned out to be really easy solution to support dark mode.

File: ui/global/global.scss [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
+}
+
+.btn, .btn.btn-default {
+  @include theme-mode {
+    color: #c8c3bc;
+    text-shadow: rgba(24,26,27,.75) 0px 1px 1px;
+    background-color: #1e2021;
+    background-image: linear-gradient(#181a1b, #26292b);
+    border-color: #3e4446 #3e4446 #43494c;
+    box-shadow: rgba(24,26,27,.2) 0px 1px 0px inset,rgba(0,0,0,.05) 0px 1px 2px
```
Comment (jazeee) [inline ui/global/global.scss:23]: I may follow up on these. I just cloned this from online.

File: ui/global/global.scss [hunk 3]
```diff
@@ -1,10 +1,33 @@
...
+  }
+}
+
 .ui-dialog {
-  background-color: $paper-color;
+  @include theme-background-color(paper-color);
   background-size: cover;
-  background-image: url(https://raw.githubusercontent.com/epic-power-rpg/roll20/main/ui/images/background.png);
+  background-image: url(https://raw.githubusercontent.com/epic-power-rpg/roll20/main/ui/images/epic-power-game-image-overlay.png);
```
Comment (jazeee) [inline ui/global/global.scss:30]: Roar.

File: ui/rollSection/rollSection.scss [hunk 4]
```diff
@@ -2,8 +2,6 @@
 
 .roll-section {
   display: flex;
-  /* Get rid of the space the sheet puts above us. */
-  margin-top: -26px;
```
Comment (jazeee) [inline ui/rollSection/rollSection.scss:6]: I cleaned up some odd styles, and addressed these issues.

File: ui/rollSection/rollSection.scss [hunk 5]
```diff
@@ -19,7 +17,7 @@
   padding: 1px 2px;
   border-width: 1px;
   border-radius: 3px;
-  background-color: $primary-background-color;
+  @include theme-background-color(primary-background-color);
```
Comment (jazeee) [inline ui/rollSection/rollSection.scss:20]: Most conversions are just adding `@include theme-` and using the same color name.

**Review by johnlamping — APPROVED** [review]
Thanks!

**Pull request merged**